enarx-keep/internal/shim-sev/src/frame_allocator.rs

```diff
@@ -353,15 +353,18 @@ impl FrameAllocator {
     }
 
     /// Map physical memory to the given virtual address
-    pub fn map_memory(
+    pub fn map_memory<T>(
```

Comment:
```suggestion
    pub fn map_memory<T: Mapper<Size4KiB> + Mapper<Size2MiB>>(
```

Reply:
For simple declarations, this is my preference. For more complex situations, us `where`.

Reply:
as you wish